quartz/src/main/java/org/quartz/listeners/BroadcastJobListener.java

```diff
@@ -37,7 +37,7 @@
  * @see #removeListener(org.quartz.JobListener)
  * @see #removeListener(String)
  *
- * @author James House (jhouse AT revolition DOT net)
+ * @author James House (jhouse AT revolution DOT net)
```

Comment:
Oops, this is an email address. I have filters for email addresses, but they don't cover AT...DOT things...